docs/API.md

```diff
@@ -311,56 +311,75 @@ for obj in uploads:
 ```
 
 <a name="get_bucket_policy"></a>
-### get_bucket_policy(bucket_name, prefix)
+### get_bucket_policy(bucket_name)
 Gets current policy of a bucket.
 
 __Parameters__
 
 |Param   |Type   |Description   |
 |:---|:---|:---|
 |``bucket_name``   | _string_  |Name of the bucket.|
-|``prefix``   |_string_    |The prefix of objects to get current policy. |
 
 __Return Value__
 
 |Param   |Type   |Description   |
 |:---|:---|:---|
-|``Policy``   | _minio.policy.Policy_   |Policy enum. Policy.READ_ONLY,Policy.WRITE_ONLY,Policy.READ_WRITE or Policy.NONE.   |
+|``Policy``    |  _string_ | Bucket policy in JSON format.|
 
 __Example__
 
 
 ```py
-# Get current policy of all object paths in bucket that begin with my-prefixname.
-policy = minioClient.get_bucket_policy('mybucket',
-                                       'my-prefixname')
+# Get current policy of all object paths in bucket "mybucket".
+policy = minioClient.get_bucket_policy('mybucket')
```

Comment:
When the bucket doesn't have a policy, this function raises minio.error.NoSuchBucketPolicy exception, before, we were returning None. It makes sense to discuss if we need to keep the old behavior or not, and document this in all cases.

Reply:
We should return `NoSuchBucketPolicy` since that would be expected now. it's a breaking change so don't expect old behavior.

Reply:
okay, then we need to document this

Reply:
You mean in the Release Notes, right? Yes.